src/pages/docs/courier/installation.md

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
Hey when previewing this page, I see the right side menu being a bit weird, with how it shows 
![Screenshot 2023-01-17 at 11 36 09 AM](https://user-images.githubusercontent.com/74224571/212775202-af6558af-40bb-402d-9eb3-7d345aaa5e14.png)

Reply:
plus when i hit upgrade on windows it takes me to the macOS one